src/components/Converters/ClassConverter.ts

```diff
@@ -70,12 +70,13 @@ export class ClassConverter extends RecordConverter {
         const interfaceRows: string[] = [];
         const TAB = SpecialCharacterHelper.TAB;
 
-        let fullName = data.name;
-        if (data.namespace) { fullName = `${data.namespace}.${fullName}`; }
+        const shortname = data.name;
```

Comment:
rename to `shortName` in accordance with `fullName`, if we want to make this an extra variable. Might be fine to just use `data.name`?